source/api/ut_runner.pkb

```diff
@@ -89,12 +89,17 @@ create or replace package body ut_runner is
     l_coverage_schema_names ut_varchar2_rows;
     l_exclude_object_names  ut_object_names := ut_object_names();
     l_include_object_names  ut_object_names;
+    l_paths                 ut_varchar2_list := ut_varchar2_list();                 
   begin
     ut_event_manager.initialize();
     begin
       ut_expectation_processor.reset_invalidation_exception();
       ut_utils.save_dbms_output_to_cache();
 
+      for i in 1..a_paths.COUNT loop
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Replace this FOR loop by a more robust WHILE one. ([plsql:ForLoopUsageCheck](https://sonarcloud.io/organizations/utplsql/rules#rule_key=plsql%3AForLoopUsageCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=utPLSQL&pullRequest=710&issues=AWSqpmjUsk71k7lwAN8c&open=AWSqpmjUsk71k7lwAN8c)

[ISSUE_KEY]: AWSqpmjUsk71k7lwAN8c